infra/integration-image/Dockerfile

```diff
@@ -34,11 +34,23 @@ RUN curl -sL https://deb.nodesource.com/setup_7.x | bash - && \
     e2fsprogs \
     iptables \
     xfsprogs \
-    xz-utils && \
+    # Speed up ISO builds with already installed reqs
+    yum \
+    yum-utils \
+    cpio \
+    rpm \
+    ca-certificates \
+    xz-utils \
+    xorriso && \
 	# Cleanup
     apt-get autoremove -y && \
     rm -rf /var/lib/apt/lists/*
 
+COPY *.repo /etc/yum/repos.d/
+RUN yum-config-manager --enable photon photon-updates
+
+RUN mkdir -p /build/rootfs
+RUN yum --installroot /build/rootfs install coreutils filesystem linux-esx e2fsprogs gzip haveged iproute2 iptables iputils logrotate net-tools openssh photon-release procps-ng sudo systemd tdnf vim systemd -y --nogpgcheck
```

Comment:
I think this will inflate the size of the bootstrap ISO significantly as we're installing many unnecessary packages into the base archive.
Why not look at using the caching mechanism that's already written into the scripts for local use?

We had enabled the drone cache option in 0.4 but over time it seemed to inflate and took a long time to unpack so it was disabled. However we can try it again in 0.5 - combined with the `make clean` call (not `make distclean` which would delete the staging builds) this would have a similar effect, maybe better as it wouldn't cause rebuild of the application-stanging.tgz and bootstrap-staging.tgz archives each time.